pages/services/kafka/2.6.0-2.2.1/notes/index.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Release notes for 2.6.0-2.2.1
+title: Release Notes
+menuWeight: 120
+model: /services/kafka/data.yml
+render: mustache
+---
+
+## Version 2.6.0-2.2.1
+
+### Updates
+- Upgrade the base dcos-commons SDK version to `0.56.1`.
+- Upgrade {{ model.techShortName }} base tech to version `2.2.1`. See [{{ model.techShortName }}'s Release Notes](https://www.apache.org/dist/kafka/2.2.1/RELEASE_NOTES.html) for details.
+- Oracle JDK is replaced by OpenJDK 8 
+
+### New Features
+- Autosuggestion available for Service Account and Secrets when launching the service from DC/OS UI
+- Support for Secure JMX
```

Comment:
Please include a link to the advanced section because there is a lot that comes with this

Reply:
Done